### ISSUE TITLE
[BUG] Kaffy don't show entities

### ISSUE THREAD

**Issue by jcmaciel** [aesmail/kaffy#197]
**Versions Used**
Kaffy: 0.9.0
Phoenix: 1.5.7
Elixir: 1.11.3

**What's actually happening?**
I've a "Boarding" schema on my app, and I firstly installed kaffy. When I go to the /admin and click on "Boardings", it's do the query, as:
![query](https://user-images.githubusercontent.com/81674/106659360-62edc880-657d-11eb-9eaf-11d37731717d.png)
Which is correct!

However, nothing is shown:
![image](https://user-images.githubusercontent.com/81674/106659208-2cb04900-657d-11eb-9718-11d5e0c92692.png)

I also defined resources, as:
```elixir
# config.exs
config :kaffy,
  otp_app: :bport,
  ecto_repo: Bport.Repo,
  router: BportWeb.Router,
  resources: &Bport.Kaffy.Config.create_resources/1

# kaffy_resources.ex
defmodule Bport.Kaffy.Config do
  def create_resources(_conn) do
    [
      boardings: [
        name: "Boardings",
        resources: [
          boarding: [schema: Bport.Boardings.Boarding]
        ]
      ],
      db_views: [
        name: "Database Views",
        resources: [
          boarding_op: [schema: Bport.DbViews.BoardOperation]
        ]
      ]
    ]
  end
end
````

It could be a misconfiguration of my part?

**What should happen instead?**
The query should be ok but kaffy also should show the result!

**Comment by aesmail** [comment]
I do actually have this issue with one of my schemas and it's driving me crazy. I haven't needed to look at it since it's one of those schemas that should never be touched "manually" and handled completely by the system. I'll dig a bit and see if there's something weird going on.

**Comment by Mdsp9070** [comment]
any updates or even a bet?

**Comment by daviaws** [comment]
I also had this problem...

![2021-03-22-160144_1366x768_scrot](https://user-images.githubusercontent.com/12686471/112043923-f37d6980-8b27-11eb-9c01-643fdf09d0f7.png)

As you can see I have data in the database.

But...

![2021-03-22-160051_1366x768_scrot](https://user-images.githubusercontent.com/12686471/112043969-0132ef00-8b28-11eb-98f7-bbccc11f942c.png)

**Comment by daviaws** [comment]
I have a bet:
The problem seems to be in the index query and some field of the entity...

I changed the custom query to be like:
`  def custom_index_query(conn, _schema, query) do
    InviteConfirmation
  end
`
And the result is:
![2021-03-22-161914_1366x768_scrot](https://user-images.githubusercontent.com/12686471/112046150-6ee01a80-8b2a-11eb-9836-8397fb01963f.png)

The pagination is bugged yet though.

**Comment by daviaws** [comment]
I found that the query without results was like:
```
#Ecto.Query<from i0 in Referral.Referreds.InviteConfirmation,
 or_where: ilike(i0.source, ^"%%"), order_by: [desc: i0.id], limit: ^100,
 offset: ^0>
 ```
 
Happens that I had source as nil. Not sure if this is the cause, but... this works:
```
def custom_index_query(conn, _schema, query) do
    query
    |> exclude(:where)
    |> do_custom_index_query(conn.query_params)
  end
```

**Comment by daviaws** [comment]
@aesmail I suggest remove the default ilike from all fields and force custom query... to avoid undesired behavior as this...

**Comment by Mdsp9070** [comment]
Thxs for this bet! I'll try this too!

**Comment by aesmail** [comment]
This issue should be fixed by #155 and #224 

If you are still facing this issue please let us know.
